media/css/firefox/nothing-personal/_browser.scss

```diff
@@ -4,226 +4,67 @@
-            background-image: url("/media/img/firefox/nothing-personal/heart-sticker.svg");
-            background-repeat: no-repeat;
-            background-size: contain;
-            filter: drop-shadow(5px 4px 13px rgba(0, 0, 0, 0.2));
-            height: 82px;
-            top: 10px;
-            right: 35px;
-            width: 90px;
-        }
-    }
+.c-browser-bar {
+    --heading-size: var(--body-sm-size);
+    background: $browser-background $browser-cool-gradient;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    border-top: $border-black;
+    border-left: $border-black;
+    border-right: $border-black;
+    border-top-left-radius: $border-radius-md;
+    border-top-right-radius: $border-radius-md;
+    padding: $spacing-sm $spacing-md;
+    min-height: 1rem; // fallback
+    min-height: var(--heading-size); // reserve space for bars without text
 
     @media #{$mq-md} {
-        margin-bottom: $layout-lg;
-
-        .c-browser-window-top {
-            &::after {
-                height: 95px;
-                bottom: 6px;
-                right: 205px;
-                top: auto;
-                width: 105px;
-            }
-        }
+        --heading-size: var(--title-3xs-size);
     }
 }
 
-// Layered background styles for first window
-.c-browser-windows {
-    padding-top: $layout-lg;
-
-    .c-layered-browser {
-        position: relative;
-        margin-bottom: $layout-lg;
-
-        .c-browser {
-            &::before {
-                content: "";
-                position: absolute;
-                width: 88%;
-                height: 99%;
-                background-color: $color-yellow-30;
-                border-radius: $border-radius-lg;
-                top: 15px;
-                right: 12px;
-                z-index: -1;
-            }
-        }
-
-        @media #{$mq-md} {
-            margin-bottom: $layout-xl + $layout-xs; // extra room for the layered windows
-            .c-browser {
-                &::before {
-                    content: none;
-                }
-            }
-        }
-
-        .c-hidden-browser {
-            display: none;
-
-            @media #{$mq-md} {
-                display: block;
-
-                &.yellow {
-                    position: absolute;
-                    right: -20px;
-                    top: 20px;
-                    z-index: 5;
-
-                    .c-browser-bar,
-                    .c-browser-content {
-                        background-color: $color-yellow-10;
-                    }
-                }
-
-                &.dark-yellow {
-                    position: absolute;
-                    top: 40px;
-                    right: -40px;
-                    z-index: 3;
-
-                    .c-browser-bar,
-                    .c-browser-content {
-                        background-color: $color-yellow-30;
-                    }
-                }
+// TODO: cross-browser checks
```

Comment:
sanity check this tomorrow (still need to remove on mobile in things section, but not sure this method is worth keeping)